src/AffineTraversal.js

```diff
@@ -10,11 +10,7 @@ class affineTraversalT {
 
   over = (f, obj) => {
     const v = this.preview(obj)
-    if (v === null) {
-      return obj
-    } else {
-      return this.set(f(v), obj)
-    }
+    return v === null ? obj : this.set(f(v), obj)
```

Comment:
Do you need to check explicitly for `null` or a cast to Falsy would do?
```suggestion
    return !v ? obj : this.set(f(v), obj)
```

Reply:
No, falsy won't work, because `false` or `0` are also falsy, an in that case we want the operation to go on.